.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: node_js
+node_js:
+  - "lts/*"
+cache:
+  npm: true
+before_install:
+  - export CHROME_BIN=/usr/bin/google-chrome
+  - export DISPLAY=:99.0
+  - sh -e /etc/init.d/xvfb start
+script:
```

Comment:
We should probably run lint as well...I think we have that on the other repo?

Reply:
Also, I can't recall if travis automatically does a build...I think it might just build the things in test:ci. Since we don't have tests for everything, we should at least build everything here also.

Reply:
good call!